### PR TITLE
Added a logo

### PR DESCRIPTION
**Pull request by aburrell**
Description
===========

Created a logo and added it to the docs and README.

Type of change
--------------
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)
* This change requires a documentation update

How Has This Been Tested?
-------------------------
Needs PR to test

### Test Configuration

* Operating system: OS X Big Sur
* Python version number: (e.g., 3.9)
* Compiler with version number: N/A
* Relevant local setup details: develop

Checklist
---------
- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``Changelog.rst``, summarising the changes
- [x] Add yourself to ``AUTHORS.rst`` and ``.zenodo.json``

### CODE REVIEW

**Review by ljlamarche — COMMENTED** [review]
Can we see the new docs generated on RTD?  I don't think I current have permission to make the build.

File: docs/apexpy.png
[the remark this answers is not in the document]
Comment (aburrell) [reply docs/apexpy.png:1]: Yup! Best idea I could come up with 😄

**Comment by aburrell** [comment]
If you click on the 'details' of the docs/readthedocs.org:apexpy test, and then on the small 'view docs' link to the right of the 'build completed' button (not the big 'view docs' button) you can see the Read the Docs build.  This link will also take you there: https://apexpy--118.org.readthedocs.build/en/118/

File: docs/conf.py [hunk 1]
```diff
@@ -35,6 +36,9 @@
 autoapi_keep_files = True
 autoapi_root = 'autoapi/generated'
 
+# The name of an image file (relative to this directory) to place at the top
+# of the sidebar.
+html_logo = os.path.join(os.path.abspath('.'), 'apexpy.png')
```
[the remark this answers is not in the document]
Comment (ljlamarche) [reply docs/conf.py:41]: I think it's supposed to be the "better" way to manage package data?  It's not too hard to implement, we can have a sidebar about it if you'd like, but I'm definitely fine dropping it for now!

**Pull request merged**